cluster-autoscaler/core/scale_down_test.go

```diff
@@ -643,7 +649,7 @@ func TestDeleteNode(t *testing.T) {
 			sd := NewScaleDown(&context, clusterStateRegistry)
 
 			// attempt delete
-			result := sd.deleteNode(n1, pods, provider.GetNodeGroup("ng1"))
+			result := sd.deleteNode(n1, pods, daemonSetPods, provider.GetNodeGroup("ng1"))
```

Comment:
If you're not adding any DS pods to any of the test cases, I'd just put an empty list here and remove the changes above.